packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -110,6 +111,8 @@ export default class PickerPlugin implements EditorPickerPluginInterface {
         }
         if (event.eventType == PluginEventType.KeyUp && !this.eventHandledOnKeyDown) {
             this.onKeyUpDomEvent(event);
+        } else if (event.eventType == PluginEventType.KeyPress) {
+            this.isCharacterValue = true;

```

Comment:
Why is this always true?

Reply:
According to spec it's because modifier keys don't send this event. Mind adding a comment? Do we know of browsers that don't do this?